staff/judith-gongora/js-arrays/fill.js

```diff
@@ -0,0 +1,44 @@
+function fill(){
+    if (!(arguments[0] instanceof Array)) throw Error(arguments[0] +' is not valid');
+    
+    if (arguments[1] == undefined){
```

Comment:
Could you loop through arguments and check if they are undefined to shorten your code?